zproject/urls.py

```diff
@@ -541,6 +540,14 @@
         zerver.views.registration.accounts_home_from_multiuse_invite,
         name='zerver.views.registration.accounts_home_from_multiuse_invite'),
 
+    # Used to generate a Zoom video call URL
+    url(r'^callback/register_zoom_user$', rest_dispatch,
+        {'GET': 'zerver.views.video_calls.register_zoom_user'}),
+    url(r'^callback/complete_zoom_user$', rest_dispatch,
+        {'GET': 'zerver.views.video_calls.complete_zoom_user'}),
+    url(r'^callback/deregister_zoom_user$', rest_dispatch,
+        {'GET': 'zerver.views.video_calls.deregister_zoom_user'}),
```

Comment:
These should not be using `rest_dispatch`.  I think you want to just pass a view name and use `require_GET` and `@zulip_login_required`

Reply:
Maybe `/callback/zoom/register` and friends.